packages/lib-classifier/src/components/Classifier/components/SubjectViewer/SubjectViewer.js

```diff
@@ -7,11 +7,13 @@ import getViewer from './helpers/getViewer'
 
 function storeMapper (stores) {
   const { active: subject, loadingState } = stores.classifierStore.subjects
-  const { onSubjectReady } = stores.classifierStore.subjectViewer
+  const { onSubjectReady, onError, loadingState: subjectReadyState } = stores.classifierStore.subjectViewer
```

Comment:
Could we get a rename for the above `loadingState` to make it extra clear the difference between the two being checked in this component? How about `subjectQueueState`?

Reply:
Sure. Is that the loading state on line 9?

Reply:
Yes, that's the one.